AssimpKit/Library/iOS-Example/iOS-Example.xcodeproj/project.pbxproj

```diff
@@ -345,17 +345,17 @@
 				SWIFT_COMPILATION_MODE = wholemodule;
 				TARGETED_DEVICE_FAMILY = "1,2";
 				VALIDATE_PRODUCT = YES;
-				VALID_ARCHS = "arm64 armv7";
 			};
 			name = Release;
 		};
 		77CDAB541DD7421600B7E342 /* Debug */ = {
 			isa = XCBuildConfiguration;
 			buildSettings = {
 				ASSETCATALOG_COMPILER_APPICON_NAME = AppIcon;
-				DEVELOPMENT_TEAM = PRJ5CHFB5D;
 				CLANG_ENABLE_MODULES = YES;
+				DEVELOPMENT_TEAM = YG7673FQ92;
 				ENABLE_BITCODE = NO;
+				"EXCLUDED_ARCHS[sdk=iphonesimulator*]" = arm64;
```

Comment:
The prebuilt `assimp` library doesn't have support for the simulator on ARM Macs yet, didn't want to rebuild the entire thing since we're all stuck with x86 laptops anyway...